packages/contentstack-import/src/utils/log.ts

```diff
@@ -26,7 +26,8 @@ export function log(entryOrMessage: MessageType, logType?: LogsType, hidden?: bo
 
 export function initLogger(config?: ImportConfig | undefined) {
   if (!logger) {
-    const basePath = pathValidator(join(config?.data ?? process.cwd(), 'logs', 'import'));
+    const basePath =
+      process.env.CS_CLI_CONFIG_PATH || pathValidator(join(config?.data ?? process.cwd(), 'logs', 'import'));
```

Comment:
@cs-raj , Please check with the team if we support environment-based configuration paths and if so, we also need to support them in the case of the export plugin. Furthermore, please verify the clone plugin.

Reply:
Removed the env option, we now only support config based and have the common var for both import and export the logs will include audit logs also.